src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java

```diff
@@ -57,43 +57,36 @@ public NavigationSideComponentObject(WebDriver driver) {
   }
 
   public void clickSearchField() {
-    wait.forElementVisible(searchInput);
-    searchInput.click();
+    waitAndClick(searchInput);
   }
 
   public void clickSearchButton() {
-    wait.forElementVisible(searchButton);
-    searchButton.click();
+    waitAndClick(searchButton);
   }
 
   public void clickBackChevron() {
-    wait.forElementVisible(backChevron);
-    backChevron.click();
+    waitAndClick(backChevron);
   }
 
   public void clickCancelButton() {
-    wait.forElementVisible(cancelSearchCaption);
-    cancelSearchCaption.click();
+    waitAndClick(cancelSearchCaption);
   }
 
   public void clickSuggestion(int index) {
-    wait.forElementVisible(searchSuggestion);
-    searchSuggestion.click();
+    waitAndClick(searchSuggestion);
   }
 
   public void clickNavListElement(int index) {
-    wait.forElementVisible(navList.get(index));
-    navList.get(index).click();
+    waitAndClick(navList.get(index));
   }
 
-  public void clickOverlay() {
+  public void tapOverlay() {
```

Comment:
This can not be verified because the only test that is using this method is currenlty unstable